Stck.Tests/StdlibTests.fs

```diff
@@ -32,6 +32,11 @@ open Stck
 [<InlineData("false true or", "true")>]
 [<InlineData("true false or", "true")>]
 [<InlineData("false false or", "false")>]
+// xor

```

Comment:
Seems to work well for all cases. Nice! :+1:

Reply:
🎉